crates/bevy_ui/src/lib.rs

```diff
@@ -39,9 +41,33 @@ pub enum UiSystem {
     Focus,
 }
 
+#[derive(Debug)]
+/// The current scale of the UI for all windows
+///
+/// ## Note
+/// This is purely about the logical scale, and can
+/// be considered like a zoom
+///
+/// This only affects pixel sizes, so a percent size will stay at that
+pub struct UiScale {
+    /// The scale to be applied
+    ///
+    /// # Example
+    ///
+    /// A scale of `2.` will make every pixel size twice as large.
```

Comment:
I think this whole 'example' section is potentially superfluous

Reply:
Potentially, I prefer to make it clear though that one needs to put in `2.` instead of `0.5` for twice the scale.